src/components/ReturnToOverviewButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function ReturnToOverviewButton() {
+  return (
+    <ButtonContainer>
+      <ReturnRoute href="/">
+        <ReturnButton>
+          <ReturnImage
+            xmlns="http://www.w3.org/2000/svg"
+            height="28"
+            viewBox="0 0 24 24"
+            width="28"
+          >
+            <path d="M0 0h24v24H0z" fill="none" />
+            <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z" />
+          </ReturnImage>
+          <ButtonText>Return</ButtonText>
+        </ReturnButton>
+      </ReturnRoute>
+    </ButtonContainer>
+  )
+}
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+  height: fit-content;
+`
+
+const ReturnRoute = styled.a`
+  text-decoration: none;
+`
+
+const ReturnButton = styled.button`
+  display: block;
+  border: none;
+  background-color: white;
```

Comment:
Why not working with CSS variables as you did in the other styling? Also, I found white to be hard to differentiate from the white background. 
![image](https://user-images.githubusercontent.com/68224104/94004834-8e75d080-fd9d-11ea-880d-fe5e578e493b.png)